### PR TITLE
fix: bump alexapy to 1.29.3

### PR DESCRIPTION
**Pull request by srescio**
tested in local dev instance that login loop is indeed fixed with the bump

addresses the following
https://gitlab.com/keatontaylor/alexapy/-/releases/v1.29.3

mapped in this repo as
https://github.com/alandtse/alexa_media_player/issues/2514
https://github.com/alandtse/alexa_media_player/issues/2515

### CODE REVIEW

**Comment by vhuynen** [comment]
Kudo @srescio,

I confirm that the alexapy **1.29.3** library fixes the login loop screen issue. To address this before the official release, you can upgrade the alexapy library in the file /homeassistant/custom_components/alexa_media/manifest.json:
`"requirements": ["alexapy==1.29.3", "packaging>=20.3", "wrapt>=1.14.0"]`
Restart Home Assistant and complete the OAuth flow to retrieve the refresh token used for API calls.

My settings HA :
Core : 2024.10.3
Supervisor : 2024.10.2
Operating System : 13.2

File: custom_components/alexa_media/manifest.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/alandtse/alexa_media_player/issues",
   "loggers": ["alexapy", "authcaptureproxy"],
-  "requirements": ["alexapy==1.29.2", "packaging>=20.3", "wrapt>=1.14.0"],
-  "version": "4.13.4"
+  "requirements": ["alexapy==1.29.3", "packaging>=20.3", "wrapt>=1.14.0"],
+  "version": "4.13.5"
```
Comment (alandtse) [inline custom_components/alexa_media/manifest.json]: Do not touch the component version. That is handled by ci.

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "alexa_media_player"
-version = "4.13.4"
+version = "4.13.5"
```
Comment (alandtse) [inline pyproject.toml]: Same as above.

**Review by alandtse — CHANGES_REQUESTED** [review]

File: custom_components/alexa_media/const.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
     PERCENTAGE,
 )
 
-__version__ = "4.13.4"
+__version__ = "4.13.5"
```
Comment (srescio) [inline custom_components/alexa_media/const.py]: just for confirmation @alandtse does it apply also to `const.py` here?

Comment (alandtse) [reply custom_components/alexa_media/const.py]: Yes. The version is handled by ci. Any mismatch will break it.

Comment (srescio) [reply custom_components/alexa_media/const.py]: i take its managed by [this action](https://github.com/alandtse/alexa_media_player/blob/dev/.github/workflows/semantic_release.yml#L79-L80)

**Comment by Wizard99it** [comment]
@srescio Sei un grande 💪🏼 Grazie, fix  4.13.5 working until now!

**Pull request merged**